Tasks/PackerBuildV1/task.json

```diff
@@ -0,0 +1,325 @@
+{
+    "id": "845fd4f4-642d-4694-8514-047948a5a556",
+    "name": "PackerBuild",
+    "friendlyName": "Build Machine Image",
+    "description": "Build machine image using Packer. This image can be used for Azure Virtual machine scale set deployment",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=845329)",
+    "category": "Deploy",
+    "releaseNotes": "- Works with cross-platform agents (Linux, macOS, or Windows)\n- Supports building images for both Windows and Linux. Supports both managed and unmanaged VM images. These images can be used to deploy Azure Virtual machine scale set.",
```

Comment:
release notes should have details of what is new in this version. You can mention that this task now supports managed disk and uses newer version of packer.

Reply:
Done.